evap/staff/tests/test_views.py

```diff
@@ -7,8 +8,8 @@
 from model_mommy import mommy
 import xlrd
 
-from evap.evaluation.models import Semester, UserProfile, Course, CourseType, \
-                                   TextAnswer, Contribution
+from evap.evaluation.models import Semester, UserProfile, Course, CourseType, TextAnswer, Contribution, Questionnaire,\
+    Question
```

Comment:
well, pycharm's defaults are not really an argument. usually IDEs are adjusted to follow the project's style, not the other way round. the previous indentation follows PEP8 which is why I tried to use it. besides, in my opinion you shouldn't deviate from a project's style just for personal preference. that said, we can agree on a new style if you want, but then apply it to the whole project.